src/lib/wallet/GoodWalletClass.js

```diff
@@ -491,11 +521,18 @@ export class GoodWallet {
    * Get transaction fee from GoodDollarReserveContract
    * @returns {Promise<boolean>}
    */
-  getTxFee(): Promise<boolean> {
-    return this.tokenContract.methods
-      .getFees(1)
-      .call()
-      .then(toBN)
+  async getTxFee(): Promise<boolean> {
+    try {
```

Comment:
1. do like balanceOf, just return balanceValue (instead of balanceValue.toNumber() )
2. really why here we don't call toNumber ? pls investigate this, it's looking strange

Reply:
why isNaN is used here ? i used it in balanceOfNative because parseInt returns NaN if input is wrong
here no parseInt used so isNaN doesn't needed

Reply:
and how about investigation abut p. 2 ? why .toNumber() is used everywhere but not in getTxFee() ? please write what you've found about this one